.github/workflows/build.yml

```diff
@@ -148,6 +148,11 @@ jobs:
           - ubuntu-latest
           # We need to use macos-15 to test with Xcode 16
           - macos-15
+        exclude:
+          - wizard: React-Native
+        include:
+          - wizard: React-Native
+            os: macos-15
```

Comment:
Running on `ubuntu-latest` will make the `pod-install` step to be skipped and the test to fail.

Reply:
Can we run the test with `pod-install -> no` on ubuntu?

Reply:
I reverted the `build.yml` change and [optionally skipped the `pod-install` check and iOS tests](https://github.com/getsentry/sentry-wizard/pull/889/files#diff-6280c9ecf15f057015e5983b97b9f4b840ac3dd3bb11ef5a3b8bd6f4d670a573).